server/test_server.py

```diff
@@ -27,3 +27,22 @@ async def test_register_app(client):
     assert response["message"]
     assert response["applications"]
     assert len(response["applications"])
+
+
+async def test_delete_app(client, applications):
+    message = {
```

Comment:
@bratxemss I suggest checking that the application exists in the database before deleting

```
    from .models import Application

    assert await Application.select().count() > 0
```